src/graph/executor/query/AppendVerticesExecutor.cpp

```diff
@@ -69,6 +71,39 @@ folly::Future<Status> AppendVerticesExecutor::appendVertices() {
       });
 }
 
+Status AppendVerticesExecutor::handleNullProp(const AppendVertices *av) {
```

Comment:
You could eliminate this operator in optimizer based on prune properties before optimizer rules.